debezium-connector-mongodb/src/test/java/io/debezium/connector/mongodb/MongoUtilIT.java

```diff
@@ -23,7 +24,7 @@ public void testGetPrimaryAddress() {
                 .build());
 
         primary.execute("shouldConnect", mongo -> {
-            ServerAddress primaryAddress = MongoUtil.getPrimaryAddress(mongo);
+            ServerAddress primaryAddress = MongoUtil.getPreferredAddress(mongo, ReadPreference.primary());
```

Comment:
Preferred address?

Reply:
Returns an address of node adhering to given Read Preference type. I'm open to better name  suggestions:)

Reply:
Apologies, was referring to the variable name. Method name is clear and consistent.

Reply:
Test is working with primary connection and the address retrieved here is indeed a primary address. I don't see a problem problem here

Reply:
Wow, 0/2, I’m not batting very high here 😅. I missed the fact that the primary was requested as it was off screen.